students/JeroenSweerts/Lesson6/mailroom4.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+donations = {}
+donations['papa'] = [100, 5, 15]
+donations['mama'] = [12, 200, 2, 66]
+donations['bompa'] = [1000]
+donations['bobonne'] = [500, 500]
+donations['onbekende'] = [1000000]
```

Comment:
The convention in Python is to use dict literal syntax when defining a new dict with some initial data:
```
donations = {
    'papa': [100, 5, 15],
    'mama': [12, 200, 2, 66],
}
```
The bracket syntax you're using is generally used for adding new data to an existing dict.